e2e-tests/test/click-position.test.ts

```diff
@@ -0,0 +1,101 @@
+import { clickElement, takeAndCompareScreenshot } from "./testing-utils";
+
+describe("m-plane position click", () => {
+  test("can get right position on click", async () => {
+    const page = await globalThis.__BROWSER_GLOBAL__.newPage();
+
+    await page.setViewport({ width: 1024, height: 1024 });
+
+    await page.goto("http://localhost:7079/click-position-test.html");
+
+    await page.waitForSelector("m-plane");
+
+    await takeAndCompareScreenshot(page);
+
+    await clickElement(page, "m-plane");
+
+    const textSelector = await page.waitForSelector("m-label");
+    const labelContent = await textSelector?.evaluate((el) => el.getAttribute("content"));
+    const parsedTitle = JSON.parse(labelContent);
+    const { x, y, z } = parsedTitle;
+
+    expect(x).toBeLessThan(0.1);
+    expect(x).toBeGreaterThan(-0.1);
+    expect(y).toBeLessThan(0.1);
+    expect(y).toBeGreaterThan(0);
+    expect(z).toEqual(0);
+
+    await takeAndCompareScreenshot(page);
+
+    await page.close();
+  }, 60000);
+
+  test("can get right position on click with offset", async () => {
+    const page = await globalThis.__BROWSER_GLOBAL__.newPage();
+
+    await page.setViewport({ width: 1024, height: 1024 });
+
+    await page.goto("http://localhost:7079/click-position-test.html");
+
+    await page.waitForSelector("m-plane");
+
+    await takeAndCompareScreenshot(page);
```

Comment:
I think the screenshots might be unnecessary for this test and could be brittle as they're likely to pick up the specificity of the coordinates. Can be removed later if they cause problems later